packages/types/src/clerk.ts

```diff
@@ -451,6 +451,16 @@ export type HandleOAuthCallbackParams = {
    * Full URL or path to navigate after an incomplete sign up.
    */
   continueSignUpUrl?: string | null;
+
+  /**
+   * Full URL or path to navigate after requesting email verification.
+   */
+  verifyEmailAddressUrl?: string | null;
+
+  /**
+   * Full URL or path to navigate after requesting phone verification.
+   */
+  verifyPhoneNumberUrl?: string | null;
```

Comment:
:question: Are these required for this fix? Seems like this is a broader API change for our components.

Reply:
It shouldn't need these, but for sign-in, using a relative URL won't work.

I want to redirect in these two cases:
- `sign-up/sso-callback` -> `sign-up/verify-email-address`
- `sign-in/sso-callback` -> `sign-up/verify-email-address`

For the first one to work, a relative URL like `../verify-email-address` is working as expected, but for the second one, the hash router will be necessary to navigate here.

Or I might be possibly missing something.

Reply:
I see, thanks for explaining. I guess the problem is the the transfer flow from sign in?

We are on the `sign-in/` "base" path and we're mounting components in there. Wonder if there's an alternative where we always go through `sign-up/` first or we can somehow reuse the `signUpUrl`. I'll leave this comment open to get feedback from an expert in our javascript repo. :pray:

Reply:
Hey @nikosdouvlis, there is no design document for this, but there is more information about the goal of this task in the ticket: https://linear.app/clerk/issue/USR-248

Do you have any concerns about these changes?